src/app/containers/PageHandlers/withLoading/index.jsx

```diff
@@ -1,40 +1,74 @@
-import React, { useEffect, useState } from 'react';
-import { bool, element } from 'prop-types';
+import React, { useEffect, useRef, useState } from 'react';
+import { bool, string, element } from 'prop-types';
 import styled from '@emotion/styled';
-import { GridWrapper, GridItemMedium } from '#app/components/Grid';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import Skeleton from './Skeleton';
+
+import { GridWrapper, GridItemLarge } from '#app/components/Grid';
 
 let timeout;
 const LoadingMain = styled.main`
   min-height: 100vh;
 `;
+const SkeletonWrapper = styled.div`
+  margin-top: ${GEL_SPACING_DBL};
+`;
+const StyledWrapper = styled.div`
+  outline: 0;
+`;
 
 const WithLoading = Component => {
   const LoadingContainer = ({ loading, ...props }) => {
-    const [showLoading, setShowLoading] = useState(false);
+    const loadingMessageRef = useRef();
+    const [showSkeleton, setShowSkeleton] = useState(false);
+    const { pageType } = props;
+
     useEffect(() => {
       if (loading) {
         timeout = setTimeout(() => {
-          setShowLoading(true);
+          if (['article', 'MAP', 'STY'].includes(pageType)) {
+            setShowSkeleton(true);
+          }
+          if (loadingMessageRef.current) {
+            loadingMessageRef.current.focus();
```

Comment:
Hey just a thought - how does the screen reader react to the skeleton? Should we focus the `VisuallyHiddenText` element, rather than the container div?

Reply:
good spot. I've updated the PR

Reply:
Noice. ✅